pkg/numaresourcesscheduler/objectstate/sched/sched.go

```diff
@@ -148,6 +154,32 @@ func DeploymentNamespacedNameFromObject(obj client.Object) (nrsv1alpha1.Namespac
 	return res, ok
 }
 
+func SchedulerNameFromObject(obj client.Object) (string, bool) {
```

Comment:
some unit-tests for this function can be useful

Reply:
Indeed, added! thanks for raising.